pkg/services/ngalert/notifier/status.go

```diff
@@ -8,15 +8,15 @@ import (
 
 // TODO: We no longer do apimodels at this layer, move it to the API.
 func (am *alertmanager) GetStatus() apimodels.GettableStatus {
-	config := &apimodels.PostableApiAlertingConfig{}
-	status := am.Base.GetStatus() // TODO: This should return a GettableStatus, for now it returns PostableApiAlertingConfig.
+	config := &apimodels.PostableUserConfig{}
+	status := am.Base.GetStatus() // TODO: This should return a GettableStatus, for now it returns PostableUserConfig.
 	if status == nil {
-		return *apimodels.NewGettableStatus(config)
+		return *apimodels.NewGettableStatus(&config.AlertmanagerConfig)
 	}
 
 	if err := json.Unmarshal(status, config); err != nil {
 		am.logger.Error("Unable to unmarshall alertmanager config", "Err", err)
 	}
 
-	return *apimodels.NewGettableStatus(config)
+	return *apimodels.NewGettableStatus(&config.AlertmanagerConfig)
```

Comment:
This does not seem to be related to the PR. Is this the fix that makes it parse the correct format?

Reply:
It is related, but not obviously so. I'll try to make it more obvious in a future PR though. Currently, `GrafanaAlertmanager.GetStatus()` just returns the raw config you initially provided to it. It was `PostableApiAlertingConfig` but now it's a `PostableUserConfig`.